runner/src/server/views/partials/summary-row.html

```diff
@@ -0,0 +1,55 @@
+{% macro summaryRow(item, notSuppliedText, changeText) %}
+<div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+        {{item.label}}
+    </dt>
+    <dd class="govuk-summary-list__value">
+        {% if item.value %}
+            {%if item.type == 'MultiInputField' %}
+                {% for itemValue in item.value%}
+                    {{itemValue}}<br>
+                {% endfor %}
+            {% elif item.type == 'NumberField' or item.type == 'WebsiteField' and 'http' not in item.value|lower %}
+                {{item.prefix}}{{item.value}}
+            {% elif item.type == 'ClientSideFileUploadField' %}
+                {% if item.value.files|length == 0 %}
+                    {{ notSuppliedText }}
+                {% else %}
+                    {% for file in item.value.files %}
+                        <div>
+                            <a class="govuk-link" href="/s3{{item.value.folderPath}}/download-file?filename={{file.Key}}&form_session_identifier={{file.FormSessionId}}">
+                                {{ file.Key }}
+                            </a>
+                        </div>
+                    {% endfor %}
+                {% endif %}
+            {% elif item.type == 'FreeTextField' %}
+                <div id={{item.name}}></div>
+                <script>
+                    // Nunjucks automatically converts any html to escaped html to prevent cross-site scripting
+                    // This method bellow safely unescapes it so it can be rendered correctly.
+                    document.getElementById('{{item.name}}').innerHTML = unescapeHtml('{{item.value}}');
```

Comment:
// Nunjucks automatically converts any html to escaped html to prevent cross-site scripting
                        // This method bellow safely unescapes it so it can be rendered correctly.